assets/scss/templates/_header.scss

```diff
@@ -41,7 +41,7 @@
 
     img {
       display: block;
-      height: auto;
+      height: 50px;
```

Comment:
Why the sudden hardcoded height? Is it related to the change in dopplerApp?

Reply:
Yes, in a meeting with @CarlosSampedro  and @santiagopaolucci it was defined that the logo has an upper padding of 18px, without lower padding in the image and that the padding remains 27.5px that currently has the header in doppler